src/JMSSerializerAdapter.php

```diff
@@ -6,8 +6,8 @@
 
 use JMS\Serializer\ArrayTransformerInterface;
 use JMS\Serializer\Context as JMSContext;
-use JMS\Serializer\DeserializationContext as JMSDeserializationContext;
-use JMS\Serializer\SerializationContext as JMSSerializationContext;
+use JMS\Serializer\DeserializationContext;
+use JMS\Serializer\SerializationContext;
```

Comment:
I did this on purpose to be clear that this is about jms and not about the liip serializer. but if you are sure its better like this i don't mind too much.

Reply:
It was not consistent. Why have the Jms alias for the context, but

```
use JMS\Serializer\SerializerInterface;
use Liip\Serializer\SerializerInterface as LiipSerializer;
```

was aliased the other way round.

Reply:
oh, indeed. so now everythink liip serializer is aliased and jms is the "default"? makes sense for me in the adapter.